src/main/webapp/help/addToTop.html

```diff
@@ -0,0 +1,4 @@
+<div>
+If checked, the user-added value will be added on top of the combo box, making it the new default value for the next build, 
+if <i>Default Choice</i> is set to <i>Top Most Value</i>.
+</div>
```

Comment:
JFYI: It's better to have text files always end with new line characters.
(Some text editors automatically add new lines at the ends)